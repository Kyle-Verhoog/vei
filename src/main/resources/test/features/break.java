// FAIL
// Java 1.3
public class Break {
  public int m(int x) {
    while (x>0) {
       x=x-1;
       if (x=42) break;
    }
    return x;
  }
}