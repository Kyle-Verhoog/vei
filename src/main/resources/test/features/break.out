PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
INT ┃ int
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
WHILE ┃ while
( ┃ (
IDENTIFIER ┃ x
> ┃ >
INTEGER_LITERAL ┃ 0
) ┃ )
{ ┃ {
IDENTIFIER ┃ x
= ┃ =
IDENTIFIER ┃ x
- ┃ -
INTEGER_LITERAL ┃ 1
; ┃ ;
IF ┃ if
( ┃ (
IDENTIFIER ┃ x
= ┃ =
INTEGER_LITERAL ┃ 42
) ┃ )
IDENTIFIER ┃ break
; ┃ ;
} ┃ }
RETURN ┃ return
IDENTIFIER ┃ x
; ┃ ;
} ┃ }
} ┃ }